docs/Anwendungskennzeichen.adoc

```diff
@@ -0,0 +1,28 @@
+= Anwendungskennzeichen
+
+In der TI verwenden viele Anwendungen KIM (und zukünftig TIM) als sicheren Transport von Daten zwischen Sender und Empfänger. Die Daten werden unterstützt durch Clientsysteme erzeugt und beim Empfänger unterstützt durch Clientsysteme angezeigt oder automatisch beim Empfänger verarbeitet. Um die Aufbereitung oder automatische Verarbeitung der Daten zu ermöglichen wurden neue Metadaten eingeführt. Diese sind:
+
+- die Dienstkennung
+- das Anwendungskennzeichen
+- der ServiceIdentifier
+
+== Dienstkennung
```

Comment:
Hier werden die Begriffe im Kontext einer KIM-E-Mail beschrieben. Wo ist der Bezug zum ATF? Wo muss das gesetzt werden etc?
Falls es nur darum geht diese Begriffe zu erklären, sollten die mE vom IG aus verlinkt werden. Das steht bestimmt auch schon woanders, oder?

Reply:
@gem-cp Können wir hier auf eine Dokumentation in der KIM-Api Beschreibung in unserem IG verweisen?

Reply:
war nur zur Erklärung